drivers/drivers.go

```diff
@@ -0,0 +1,98 @@
+// Package drivers abstracts different object storages, such as local, s3, ipfs
+package drivers
+
+import (
+	"crypto/tls"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/golang/glog"
+	"github.com/livepeer/go-livepeer/net"
+)
+
+// Storages configured at start (used by broadcaster)
+var Storages []OSDriver
+
+// ExternalStorage own external storage, if configured
+var ExternalStorage OSDriver
+
+// LocalStorage single instance of local storage
+var LocalStorage *localOS
+
+// OSDriver common interface for Object Storage
+type OSDriver interface {
+	SaveData(jobID int64, streamID, name string, data []byte) (*net.TypedURI, error)
+	// IsExternal
+	IsExternal() bool
+	// GetInfo return info about object storage need to be able to write this storage
+	GetInfo() *net.OSInfo
+	// EvictManifestData removes memory cache for stream
+	EvictManifestData(mid string)
```

Comment:
If we instantiate a driver per job, then we can have an `Evict()` method without any parameters. This also allows us to reuse S3 policies.

In fact, Evict wouldn't strictly be necessary (right now), since in-memory storage would be automatically GC'd once the job completes, but it's generally a useful API to have.

Reply:
We do not instantiate driver per job, that was the reason for OSSession removal.